.husky/commit-msg

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npx --no-install commitlint --edit 
\ No newline at end of file
+npx --no-install commitlint --edit "$1"
```

Comment:
What is the role of "$1"?

Reply:
with or without $1 almost same:
https://github.com/conventional-changelog/commitlint/issues/3048

$1 is the args[1] of the command, which is the commit message text.

but $1 is from the official doc:
https://typicode.github.io/husky/#/?id=husky_git_params-ie-commitlint-